### ISSUE TITLE
"errorMessage": "Missing required key 'TableName' in params"

### ISSUE THREAD

**Issue by felipe1982** [gabehollombe-aws/step-functions-workshop#4]
```node
{
  "errorType": "MissingRequiredParameter",
  "errorMessage": "Missing required key 'TableName' in params",
  "trace": [
    "MissingRequiredParameter: Missing required key 'TableName' in params",
    "    at ParamValidator.fail (/var/runtime/node_modules/aws-sdk/lib/param_validator.js:50:37)",
    "    at ParamValidator.validateStructure (/var/runtime/node_modules/aws-sdk/lib/param_validator.js:61:14)",
    "    at ParamValidator.validateMember (/var/runtime/node_modules/aws-sdk/lib/param_validator.js:88:21)",
    "    at ParamValidator.validate (/var/runtime/node_modules/aws-sdk/lib/param_validator.js:34:10)",
    "    at Request.VALIDATE_PARAMETERS (/var/runtime/node_modules/aws-sdk/lib/event_listeners.js:126:42)",
    "    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:106:20)",
    "    at callNextListener (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:96:12)",
    "    at /var/runtime/node_modules/aws-sdk/lib/event_listeners.js:86:9",
    "    at finish (/var/runtime/node_modules/aws-sdk/lib/config.js:379:7)",
    "    at /var/runtime/node_modules/aws-sdk/lib/config.js:397:9"
  ]
}
```
I got this error when trying to tie in the `SubmitApplication` lambda on [Starting our workflow when a new application is submitted](https://step-functions-workshop.go-aws.com/50_fleshing_out_our_workflow/10_starting_workflow_execution_via_api.html)

**Comment by felipe1982** [comment]
submit.js looks like this

```js
'use strict';
const REGION = process.env.REGION
const ACCOUNTS_TABLE_NAME = process.env.ACCOUNTS_TABLE_NAME
const APPLICATION_PROCESSING_STEP_FUNCTION_ARN = process.env.APPLICATION_PROCESSING_STEP_FUNCTION_ARN

const AWS = require('aws-sdk')
AWS.config.update({region: REGION});

const dynamo = new AWS.DynamoDB.DocumentClient();
const stepfunctions = new AWS.StepFunctions();

const AccountApplications = require('./AccountApplications')(ACCOUNTS_TABLE_NAME, dynamo)

const submitNewAccountApplication = async (data) => {
    const { name, address } = data
    const application = await AccountApplications.create({ name, address, state: 'SUBMITTED' })
    return application
}

const startStateMachineExecution = (application) => {
    const params = {
        "input": JSON.stringify({ application }),
        "name": `ApplicationID-${application.id}`,
        "stateMachineArn": APPLICATION_PROCESSING_STEP_FUNCTION_ARN
    }
    return stepfunctions.startExecution(params).promise()
}

module.exports.handler = async(event) => {
    let application
    try {
        application = await submitNewAccountApplication(event)
        await startStateMachineExecution(application)
        return application
    } catch (ex) {
        if (application !== undefined) {
            await AccountApplications.delete(application.id)
        }

        console.error(ex)
        console.info('event', JSON.stringify(event))
        throw ex
    }
}

```

**Comment by felipe1982** [comment]
Looks like the correct code is here

https://github.com/gabehollombe-aws/step-functions-workshop/blob/master/serverless_template/account-applications/submit.js

**Comment by felipe1982** [comment]
perhaps fixed in  d474a3f

**Comment by gabehollombe-aws** [comment]
Thanks for this, @felipe1982 !  I've just pushed f68b1a which should fix this in the website content.
